docs/reference/api/media.md

```diff
@@ -0,0 +1,153 @@
+|`title` | String | A "sortified" version of the file name (case-insensitive) |
+|`type` | String | 'attachment' |
+|`trashDocIds` | Array | An array of string IDs that represent image or file documents using this attachment in the trash |
+|`updatedAt` | Date | An [ISO date string](https://en.wikipedia.org/wiki/ISO_8601) of the document's last update date and time|
+|`width` | Number | Width of the image file, if an image |
+
+```json
+// This is a PNG upload with filename `blue-box.png`
+{
+  "_id": "ckj0akbxa003vp39kfbxgb8zg",
+  "group": "images",
+  "createdAt": "2020-12-22T17:58:11.314Z",
+  "name": "blue-box",
+  "title": "blue box",
+  "extension": "png",
+  "type": "attachment",
+  "docIds": [],
+  "trashDocIds": [],
+  "length": {
+    "dev": 16777220,
+    "mode": 33188,
+    "nlink": 1,
+    "uid": 501,
+    "gid": 20,
+    "rdev": 0,
+    "blksize": 4096,
+    "ino": 140102237,
+    "size": 10497,
+    "blocks": 24,
+    "atimeMs": 1608659891038.2366,
+    "mtimeMs": 1608659891038.5925,
+    "ctimeMs": 1608659891038.5925,
+    "birthtimeMs": 1608659891038.2366,
+    "atime": "2020-12-22T17:58:11.038Z",
+    "mtime": "2020-12-22T17:58:11.039Z",
+    "ctime": "2020-12-22T17:58:11.039Z",
+    "birthtime": "2020-12-22T17:58:11.038Z"
+  },
+  "md5": "630eeaaecd0bdc07c4a82eeca4c07588",
+  "width": 600,
+  "height": 106,
+  "landscape": true,
+  "ownerId": "ckhdsd0hk0003509kchzbdl83",
+  "_urls": {
+    "max": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.max.png",
+    "full": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.full.png",
+    "two-thirds": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.two-thirds.png",
+    "one-half": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.one-half.png",
+    "one-third": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.one-third.png",
+    "one-sixth": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.one-sixth.png",
+    "original": "https://example.net/uploads/attachments/ckj0akbxa003vp39kfbxgb8zg-blue-box.png"
+  }
+}
+```
+
+## `POST: /api/v1/@apostrophecms/attachment/crop`
+
+**Authentication required.**
+
+This route uploads a new, cropped version of an existing image. The `crop` object is appended to the `crops` array property of the attachment document.
+
+The `body` of the request should include:
+
+| Property | Format | Description |
+|----------|------|-------------|
+|`_id` | String | The `_id` property of an existing image attachment document |
+|`crop` | Object | An object containing `top`, `left`, `width` and `height` properties |
+
+The newly uploaded image file with have a file name using the `crop` properties as well as the `_id`, `name`, and file extension of the original attachment, formatted as `{_id}-{name}.{top}.{left}.{width}.{height}.{extension}`
```

Comment:
That would be correct for the original. Also show how this gets applied to the various configured sizes too.

Reply:
Note that the crop coordinates in the filenames for cropped sizes are the same as the crop coordinates for the original, they are not scaled. (Otherwise it would be very difficult to relate them to each other.)